### PR TITLE
Ag mfg - k8s observability README

### PR DESCRIPTION
**Pull request by zaidmohd**
PR includes Agora manufacturing k8s observability README guidelines.

### CODE REVIEW

**Comment by fcabrera23** [comment]
@zaidmohd  could you please rebase this branch based on the latest canary branch? I've added the configurations to get the Vale linitn check

**Review by janegilring — APPROVED** [review]
LGTM, with some suggestions to consider:

- Change "Contoso Motor's" to "Contoso Motors'" in the overview section for consistency.
- Consider rephrasing "These dashboards allow them to gain valuable insights into their infrastructure's health and performance, identify trends, and make informed decisions" to "These dashboards enable valuable insights into their infrastructure’s health and performance, facilitate trend identification, and support informed decision-making."
- In the Grafana Dashboards section, change "Author's efforts" to "authors' efforts."

File: docs/azure_jumpstart_ag/manufacturing/contoso_motors/k8s_infra_observability/_index.md [hunk 1]
```diff
@@ -11,4 +11,88 @@ technologyStack:
...
+
+## Access the dashboards
+
+As an Operations team member at Contoso, you will have access to the Grafana dashboards for infrastructure observability. These dashboards provide a comprehensive view of the health, performance, and metrics of the cloud to edge infrastructure. To access the Grafana dashboards follow the below steps.
+
+- Connect to the Client VM _Ag-VM-Client_ using the instructions in the [deployment guide](../deployment/#connecting-to-the-agora-client-virtual-machine).
+
+- Open the Edge browser, expand Grafana in the Favorites Bar and select `Grafana`.
+
+    ![Screenshot showing accessing Grafana](./img/grafana.png)
```
Comment (sebassem) [inline docs/azure_jumpstart_ag/manufacturing/contoso_motors/k8s_infra_observability/_index.md:56]: We need to update this screenshot as the frontend bookmark is changed

**Review by sebassem — APPROVED** [review]
LGTM

Just one screenshot to update

**Pull request closed without merging**